crap-advisor-frontend/src/core/components/Map/index.jsx

```diff
@@ -2,34 +2,32 @@ import React, {useEffect, useState} from "react";
 import "./index.css"
 import {MapContainer, TileLayer, ZoomControl} from "react-leaflet";
 import RestroomMarker from "../RestroomMarker";
+import {apiUrl} from "../../../app/constants";
 
-const Map = () => {
-    const apiUrl = "http://localhost:8082"
+const Map = ({selectRestroom}) => {
     const defaultPosition = [53.21176, 50.18394]
     const defaultZoom = 13
     const [restrooms, setRestrooms] = useState([])
 
     useEffect(() => {
         fetch(apiUrl + "/restroom/all")
-        .then((response) => response.json())
-        .then((data) => {
-            setRestrooms(data)
-            console.log(`Received ${data.length} restrooms`)
-        })
+            .then((response) => response.json())
+            .then((data) => {
+                setRestrooms(data)
+                console.log(`Received ${data.length} restrooms`)
```

Comment:
Please, remove logs from code. It's only for debug

Reply:
Yeah sure